### DISCUSSION TITLE
Wallet has been hack

### DISCUSSION THREAD

**Discussion by Kaushal12v** [community/community#64387] [Programming Help] [answered]
### Body

Hello there today I have met white Aaj hacker he gave clean my wallet (Fake_Phishing2105) it was his Bnb name and he sent all token to 0x00006da0e72b377e4be66cfb4522ce0786fc0000 this address please carefully click on the all airdrop links 
And is there any way to recover my token or is there any way to block him please suggest

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
Crypto scams are very common and you just need to avoid it. All transactions are final and cannot be sent back. It is the blockchain after all. Also, do not click on airdrop links unless it’s a verified project and always use a burner wallet!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Kaushal12v 👋🏼  We really appreciate you flagging this.  If any of this occurred on GitHub, the best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam on GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
